bears/go/GoVetBear.py

```diff
@@ -25,5 +25,130 @@ class GoVetBear:
     CAN_DETECT = {'Unused Code', 'Smell', 'Unreachable Code'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
-        return 'vet', filename
+    def create_arguments(filename, file, config_file,
+                         assembly_declarations: bool=False,
+                         assign: bool=False,
+                         atomic: bool=False,
+                         boolean: bool=False,
+                         build_tags: bool=False,
+                         invalid_cgo: bool=False,
+                         unkeyed_composite_literals: bool=False,
+                         copying_locks: bool=False,
+                         http_response: bool=False,
+                         lost_cancel: bool=False,
+                         methods: bool=False,
+                         nil_function: bool=False,
+                         printf: bool=False,
+                         range_loops: bool=False,
+                         shadow_variables: bool=False,
+                         shifts: bool=False,
+                         struct_tags: bool=False,
+                         tests_and_docs: bool=False,
+                         unreachable: bool=False,
+                         unsafe_pointers: bool=False,
+                         unused_result: bool=False):
+        """
+        :param assembly_declarations:
```

Comment:
Also add a deascription here of what the bear does.

Reply:
This is already done in the class docstring ;)